api/nntrainer-api-common.h

```diff
@@ -53,6 +53,10 @@ typedef enum {
   ML_TRAIN_LAYER_TYPE_RNNCELL = 22,  /**< RNN Cell Layer type (Since 7.0) */
   ML_TRAIN_LAYER_TYPE_ZONEOUTLSTMCELL =
     23, /**< ZoneoutLSTM Cell Layer type (Since 7.0) */
+  ML_TRAIN_LAYER_TYPE_ATTENTION = 24,     /**< Attention Layer type */
```

Comment:
The enumerations of attention type is already added in #1990.
How about rebase onto this pr?

Reply:
okay thank you.